src/handlebar-helpers.js

```diff
@@ -42,7 +42,25 @@ const helpers = {
       return x + y;
     }
     return -1;
-  }
+  },
+  substring: function (data, startIndex, endIndex) {
+    if ( typeof data  === 'string' && typeof startIndex === 'number' && typeof endIndex === 'number') {
+      return data.substring(startIndex, endIndex);
+    }
+    if ( typeof data  === 'string' && typeof startIndex === 'number') {
+      return data.substring(startIndex);
+    }
+    return ''
+  },
+  includes: function (arr, valueToFind, fromIndex) {
+    if ( Array.isArray(arr) && valueToFind !== 'undefined' && typeof fromIndex === 'number') {
+        return arr.includes(valueToFind, fromIndex)
+    }
+    if ( Array.isArray(arr) && typeof valueToFind !== 'undefined') {
+        return arr.includes(valueToFind)
+    }
+    return false
```

Comment:
minimze code and add semicolons
```suggestion
  includes: function (arr, valueToFind, fromIndex) {
    fromIndex = ( typeof fromIndex === 'number' ) ? fromIndex : undefined;
    if ( Array.isArray(arr) && valueToFind !== 'undefined') {
        return arr.includes(valueToFind, fromIndex);
    }
    return false;
```

Reply:
It makes sense, updating code as suggested ....